packages/opensrp-server-service/src/serviceClass.ts

```diff
@@ -151,10 +158,7 @@ export class OpenSRPService<PayloadT extends object = Dictionary> {
    * @param {function() | string} accessTokenCallBack - received access token
    */
   public static async processAcessToken(accessTokenCallBack: GetAccessTokenType | string) {
-    if (typeof accessTokenCallBack === 'function') {
-      return (await accessTokenCallBack()) as string;
-    }
-    return accessTokenCallBack;
+    return processAcessToken(accessTokenCallBack);
```

Comment:
nitpick: why did this change? does not seem like it had to.

Reply:
@p-netm I just refactored the code and exported `processAccessToken`

Reply:
yeah I see that, I was just saying it seems the previous code was still usable i.e. being imported as a static member of the service class, maybe if we had some guarantees on tree shaking, then I think your change would have a significant advantage.

Anyway this as mentioned is nitpick-ish.

Reply:
@p-netm I get you now...I did not see that it is declared as static when I was making the change